packages/nextcloud/build.yaml

```diff
@@ -18,3 +18,10 @@ targets:
             - 'factory .*\.fromJson\(Map<String, dynamic> json\) => jsonSerializers\.deserializeWith\(serializer, json\)!;'
             - 'Map<String, dynamic> toJson\(\) => jsonSerializers\.serializeWith\(serializer, this\)! as Map<String, dynamic>;'
             - 'static BuiltSet<.*> get values => _\$.*Values;'
+          overrides:
+            news.openapi.json:
+              experimental: true
+            notes.openapi.json:
+              experimental: true
+            uppush.openapi.json:
+              experimental: true
```

Comment:
But here it is just the file name. It should be the whole path relative to the root of the package.

Reply:
Should I just drop the regex altogether and always require full paths?

Reply:
Yeah that is better to avoid future problems.

Reply:
I doubt that this will lead to a lot of duplicate configuration